xgcm/grid.py

```diff
@@ -1388,6 +1387,9 @@ def interp_like(self, array, like):
             Source data array with updated positions along axes matching with target array
         """
 
+        boundary = self._parse_axes_kwargs(boundary)
+        fill_value = self._parse_axes_kwargs(fill_value)
+
```

Comment:
Using `interp_axes` and `interp_axes.append(axname)` results in a type of double interpolation, where array gets interpolated on to the first axis, and then as the loop continues to the second axis, because axname is appended to interp_axes, the array gets interpolated on to the first axis a second time. The correct output (from the first interpolation) is overwritten as result.

Reply:
I think you could just move the `interp` line outside the for loop? Or provide `axname` instead of `interp_axes` so you interp one axis at a time as you loop

Reply:
Right, so moving the interp line outside the for loop worked, thanks! 😅